setup.py

```diff
@@ -4,31 +4,35 @@
 
 def parse_requirements(requirements):
     with open(requirements) as fp:
-        return [line.strip('\n') for line in fp if line.strip('\n') and not line.startswith('#')]
+        return [
+            line.strip("\n")
+            for line in fp
+            if line.strip("\n") and not line.startswith("#")
+        ]
 
 
 HERE = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(HERE, 'README.md'), encoding='utf-8') as fp:
+with open(os.path.join(HERE, "README.md"), encoding="utf-8") as fp:
     long_description = fp.read()
 
 setup(
-    name='osf_pigeon',
-    version='0.0.9',
-    description='A utility for archiving osf storage projects at archive.org',
+    name="osf_pigeon",
+    version="0.0.9",
```

Comment:
I say we go ahead and bump to 0.1.0 or 0.0.10. The changes here are too big to keep the version the same.